### DISCUSSION TITLE
Should we migrate building and contributing part from README.md to robolectric.github.io?

### DISCUSSION THREAD

**Discussion by utzcoz** [robolectric/robolectric#7172] [Q&A]
There is a new page about building Robolectric(http://robolectric.org/building-robolectric/) at robolectric.github.io. There are many things we can add more for building and contributing. What about migrating those things from README.md to robolectric.github.io now? The only thing left in this repository are code of conduct. cc /@hoisie.

**Comment by hellosagar** [comment]
just curious to ask, is the robolectric website is generated by the .md file, or need to put extra effort to maintain the website too? like I think `mockk` have pretty good sync between their [readme](https://github.com/mockk/mockk) and [website](https://github.com/mockk/mockk)

**Comment by hoisie** [comment]
The Robolectric website is automatically generated from the contents of 
https://github.com/robolectric/robolectric.github.io. It uses Jekyll to convert markdown into HTML.

**Reply by hellosagar** [reply]
So, robolectric is already using that, but why the source code repo and documentation repo is different. If it's the same repo and utilizes the same markdown file then it will take less effort to maintain.
isn't it? just throwing my observation, as `mockk` is doing

**Reply by utzcoz** [reply]
They are not same. The robolectric.github.io has its markdown files, and those markdown files instead of files in robolectric repository will be rendered to website by [Jekyll](https://jekyllrb.com/). The website of Robolectric is more complicated than mockk, because it also contains massive blog post about important things and javadoc for every important release.

**Reply by utzcoz** [reply]
@hoisie The reason I open this discussion is that you have created more documentation on robolectric.github.io recent days, and we can consider moving current documentation on robolectric repo to robolectric.github.io. All palces are good to me, but we should keep consistent. There are many places we can find documentation of Robolectric, main repo's README.md and related markdown files, official website, main repo's WiKi. IMO we can leave WiKi for maintainers only, and combine main repo's documentation and official website's documentation to one place for users and contributors.
